route4me-csharp-sdk/Route4MeSDKLibrary/Route4MeManagerV5.cs

```diff
@@ -248,12 +249,16 @@ public AddressBookContact[] GetDepotsFromAddressBook(out ResultResponse resultRe
         {
             contactParams.PrepareForSerialization();
 
-            return GetJsonObjectFromAPIAsync<AddressBookContact>(contactParams,
+            var result = GetJsonObjectFromAPIAsync<AddressBookContact>(contactParams,
                 R4MEInfrastructureSettingsV5.ContactsAddNew,
                 HttpMethodType.Post,
                 null,
                 true,
                 false);
+
+            return Task.FromResult(
```

Comment:
What is the purpose of this change? I would recommend to avoid unnecessary memory traffic here. You can just return result or revert changes.

Reply:
Some API 5 endpoints do not return any responses, but we can extract job ID from the header if request is async. Then we can retrieve this job to check its state. so I added string variable for job id in tuple:
private async Task<Tuple<T, ResultResponse, string>> GetJsonObjectFromAPIAsync<
i.e. final function returns 3 variables in the tuple, not 2 as before. In order to preserve users' experiences with the functions, I decided to make little change inside and not to change response code for users. That was done for the functions, which return usual response but not job ID in the header.